### PR TITLE
PWM setting params

### PR DESCRIPTION
**Pull request by LorenzMeier**
This allows to set PWM min, max etc by using the new ```p:PWM_MIN``` syntax for the wpm command.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
@tumbili Ready for on hardware testing. Please use the Caipi config, you have PWM_MIN and MAX params in the param default group if you flash from the console and in the PWM Output group if you flash the binary from this branch via QGC.

**Comment by tumbili** [comment]
@LorenzMeier Doing this today in the evening.

**Comment by tumbili** [comment]
@LorenzMeier Tested, seems to work. Let me try again tomorrow after the flight once more.

**Comment by tumbili** [comment]
@LorenzMeier This is tested intensively! The functionality is working well, tested min, max and disarmed. Just two things I wanted to mention:
1) We should adjust the limit pwm values in drv_pmw_output.h
I think Remo sent some specs lately of how hight they want to go up with their PWM.

2) The default PWM_DISARMED is set to 0, is that intentional?

**Comment by LorenzMeier** [comment]
Yes. Disarmed defaults to off. Could you push a commit with the limit changes?

**Comment by tumbili** [comment]
@LorenzMeier I slightly adjusted pwm_max, the rest was ok. I also set PWM_DISARMED in the caipi config. Rebased on release and ready to go.
